code/ai/aicode.cpp

```diff
@@ -11523,7 +11508,7 @@ void render_wing_phantoms(object *objp)
 
 	for (i=0; i<32; i++)
 		if (Debug_render_wing_phantoms & (1 << i)) {
-			get_absolute_wing_pos(&goal_point, objp, i, 0);
+			get_absolute_wing_pos(&goal_point, objp, i, shipp->wingnum, false);
```

Comment:
Since `autopilot` defaults to false, you don't need to specify it here.  You can if you want, of course.

Reply:
That (and the one below) is actually the `formation_object` flag, this was the '0' previously but I changed to a bool.

Reply:
Oh, whoops. >.>